docs/specifications/tests/Nameserver-TP/nameserver10.md

```diff
@@ -1,99 +1,168 @@
 
-[RFC 6891][RFC 6891#6.1.3], section 6.1.3,
-states that if a nameserver has implemented
-EDNS but has not implemented the version level of the request, then it
-MUST respond with RCODE "BADVERS". Only version "0" has been defined for
+[RFC 6891][RFC 6891#section-6.1.3], section 6.1.3, states that if a nameserver
+has implemented EDNS but has not implemented the version level of the request,
+then it MUST respond with RCODE "BADVERS". Only version "0" has been defined for
 EDNS.
 
-Note that RCODE "BADVERS" is an extended RCODE which is calculated from
-the combination of the normal RCODE field in the DNS package header
-([RFC 1035][RFC 1035#4.1.1], section 4.1.1) and the OPT record
-EXTENDED-RCODE field ([RFC 6891][RFC 6891#6.1.3], section 6.1.3). Also see
-[IANA RCODE Registry].
+Note that RCODE "BADVERS" is an extended RCODE which is calculated from the
+combination of the normal RCODE field in the DNS package header
+([RFC 1035][RFC 1035#section-4.1.1], section 4.1.1) and the OPT record
+EXTENDED-RCODE field ([RFC 6891][RFC 6891#section-6.1.3], section 6.1.3). Also
+see [IANA RCODE Registry].
+
 
 ## Scope
 
-It is assumed that *Child Zone* has been tested by [Basic04]. This test
-case will set DEBUG level on messages for non-responsive name servers.
+It is assumed that *Child Zone* has been tested by [Basic04] and [Nameserver02].
+Issues covered by [Basic04] (basic name server issues) or [Nameserver02] (basic
+EDNS issues) will no result in messagess from this test case.
+
 
 ## Inputs
 
 * "Child Zone" - The domain name to be tested.
 
-## Ordered description of steps to be taken to execute the test case
+## Summary
+
+* Only relevant for a zone whose name servers correctly support EDNS, version 0.
+
+Message Tag outputted         | Level   | Arguments         | Description of when message tag is outputted
+:-----------------------------|:--------|:------------------|:--------------------------------------------
+N10_NO_RESPONSE_EDNS1_QUERY   | WARNING | ns_ip_list        | Response when EDNS ver=0, but not when 1.
+N10_UNEXPECTED_RCODE          | WARNING | ns_ip_list, rcode | Unexpected RCODE value when EDNS ver=1.
+N10_EDNS_RESPONSE_ERROR       | WARNING | ns_ip_list        | Expected RCODE value when EDNS ver=1, but error in response.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+
+## Test procedure
+
+1. Create the following empty sets:
 
-1. Created an SOA query for the *Child Zone* with an OPT record with
-   EDNS version set to "1" and no other EDNS options or flags.
+  1. Name server IP ("No Response EDNS1 Query").
+  2. Name server IP and associated RCODE ("Unexpected RCODE").
+  3. Name server IP ("EDNS Response Error").
```

Comment:
Indentation is not properly set (see the [file's output](https://github.com/zonemaster/zonemaster/blob/134c05b1bcb734def42c9575ad0cb8f2e3ef0970/docs/specifications/tests/Nameserver-TP/nameserver10.md#test-procedure))

Reply:
Fixed (I hope).

Reply:
this looks good